privacyidea/lib/token.py

```diff
@@ -732,15 +740,14 @@ def get_all_token_users():
 
     for tokenobject in tokenobject_list:
         user_info = {}
-        if tokenobject.token.user_id and tokenobject.token.resolver:
-            user_info = get_user_info(tokenobject.token.user_id,
-                                      tokenobject.token.resolver)
+        if tokenobject.token.first_owner:
+            user_info = get_user_info(tokenobject.token.first_owner.user_id,
+                                      tokenobject.token.first_owner.resolver)
 
-        if tokenobject.token.user_id and len(user_info) == 0:
-            user_info['username'] = u'/:no user info:/'
+            if len(user_info) == 0:
+                user_info['username'] = u'/:no user info:/'
 
-        if user_info:
-            tokens[tokenobject.token.serial] = user_info
+        tokens[tokenobject.token.serial] = user_info
 
     return tokens
```

Comment:
I was curious because I didn't encounter ``get_all_tokens_users`` before :-) It seems it is only used in the tests. Likewise, ``get_user_info`` is only used by ``get_all_tokens_users`` and in the tests. What do you think about removing them?

Reply:
This was intended also for subscription purposes or statistics purposes, like "how many tokens with users are there"?

It could be that it was used and the use case was changed a bit.
It could however be, that it could be used again...?

So I am a bit hesitant to remove this. (Although it usually makes no sense to have unused code)

Reply:
In subscriptions.py we have get_users_with_active_tokens.

I will remove this! (both)

Reply:
done in 2ae64abf